### PR TITLE
Add how to generate admin menu

### PR DESCRIPTION
**Pull request by harikt**
Forgive me for this may not be proper. But it is good to have than wondering how to generate the menu.

### CODE REVIEW

File: docs/cookbook/entities/configure-admin-menu.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
+How to configure backend menu
+===============================
```
Comment (loic425) [inline docs/cookbook/entities/configure-admin-menu.rst]: It seems to have to much "=". It needs to be identical with the title length.

File: docs/cookbook/entities/configure-admin-menu.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            ->setLabel('sylius.ui.content')
+        ;
+        $customer->addChild('backend_article', ['route' => 'app_backend_article_index'])
+            ->setLabel('app.ui.articles')
+            ->setLabelAttribute('icon', 'newspaper');
+        return $customer;
+    }
+
+    
+Below is the git diff once you make the change.
```
Comment (loic425) [inline docs/cookbook/entities/configure-admin-menu.rst]: I am not fan with a git diff on doc. I think we can improve this part.

File: docs/cookbook/entities/map.rst.inc [hunk 3]
```diff
@@ -1,3 +1,4 @@
 * :doc:`/cookbook/entities/first-resource`
 * :doc:`/cookbook/entities/manage-your-entity`
 * :doc:`/cookbook/entities/configure-your-routes`
+* :doc:`/cookbook/entities/configure-admin-menu`
```
Comment (loic425) [inline docs/cookbook/entities/map.rst.inc]: you also needs to add this entry on menu (cf Travis) to access to this cookbook.

**Comment by loic425** [comment]
@harikt Globally, you're right, this is a must-have cookbook 👍

File: docs/cookbook/entities/configure-admin-menu.rst (hunk 1 above)
Comment (harikt) [reply docs/cookbook/entities/configure-admin-menu.rst]: Oh yes. I missed it.

File: docs/cookbook/entities/configure-admin-menu.rst (hunk 2 above)
Comment (harikt) [reply docs/cookbook/entities/configure-admin-menu.rst]: ok, we can remove it.

File: docs/cookbook/entities/map.rst.inc (hunk 3 above)
Comment (harikt) [reply docs/cookbook/entities/map.rst.inc]: Not sure I get you here. Can you point where to add ? Not seeing anything on travis.yml .

**Comment by harikt** [comment]
One question regarding the `App\Form\Type\ArticleType` . This class is not there.

```
# config/packages/sylius_resource.yaml

    sylius_resource:
        resources:
            app.article:
                classes:
                    model: App\Entity\Article
                    # form: App\Form\Type\ArticleType
            app.oauth_client:
                classes:
                    model: App\Entity\OAuth\Client
```

and if it is commented out things will work. Is that purposeful to be kept left ?

**Comment by harikt** [comment]
Also is there a way to fix the edit buttons on site. They all give 404 .

https://github.com/Monofony/SymfonyStarter/blob/origin/master/docs/index.rst

vs

https://github.com/Monofony/SymfonyStarter/blob/master/docs/index.rst

File: docs/cookbook/entities/map.rst.inc (hunk 3 above)
Comment (harikt) [reply docs/cookbook/entities/map.rst.inc]: Oh, I found it. Adding.

Comment (loic425) [reply docs/cookbook/entities/map.rst.inc]: You need to add this page on this [tree](https://github.com/Monofony/SymfonyStarter/blob/master/docs/cookbook/index.rst#entities).

```
.. toctree::
    :hidden:

    entities/first-resource
    entities/manage-your-entity
    entities/configure-your-routes
    entities/configure-admin-menu
```

Comment (harikt) [reply docs/cookbook/entities/map.rst.inc]: Thank you. Already added :) . Found it a few mins before via search.

**Comment by loic425** [comment]
> One question regarding the `App\Form\Type\ArticleType` . This class is not there.
> 
> ```
> # config/packages/sylius_resource.yaml
> 
>     sylius_resource:
>         resources:
>             app.article:
>                 classes:
>                     model: App\Entity\Article
>                     # form: App\Form\Type\ArticleType
>             app.oauth_client:
>                 classes:
>                     model: App\Entity\OAuth\Client
> ```
> 
> and if it is commented out things will work. Is that purposeful to be kept left ?

You don't have too but then it uses a [default form type](https://github.com/Sylius/SyliusResourceBundle/blob/master/src/Bundle/Form/Type/DefaultResourceType.php). I don't think it's a good way to use it (Sylius never did that) We need to add this article form type on the cookbook.

**Comment by loic425** [comment]
> Also is there a way to fix the edit buttons on site. They all give 404 .
> 
> https://github.com/Monofony/SymfonyStarter/blob/origin/master/docs/index.rst
> 
> vs
> 
> https://github.com/Monofony/SymfonyStarter/blob/master/docs/index.rst

Can you open an issue?

File: docs/cookbook/entities/first-resource.rst [hunk 4]
```diff
@@ -46,6 +46,44 @@ As an example we will take an **Article entity**.
...
+
+.. code-block:: php
+
+    // src/Form/Type/ArticleType.php
+
+    namespace App\Form\Type;
+
+    use Symfony\Component\Form\AbstractType;
+    use Symfony\Component\Form\Extension\Core\Type\TextType;
+    use Symfony\Component\Form\Extension\Core\Type\TextareaType;
```
Comment (loic425) [inline docs/cookbook/entities/first-resource.rst]: It's not used here. My bad I think.

Comment (harikt) [reply docs/cookbook/entities/first-resource.rst]: lol. I added one for testing. I will remove it.

**Comment by harikt** [comment]
I think I made some problems with the rebase. I will send a different PR.

**Comment by harikt** [comment]
https://github.com/Monofony/SymfonyStarter/pull/184

**Pull request closed without merging**